test/integration/util/util.go

```diff
@@ -1157,6 +1157,8 @@ func NextPodOrDie(t *testing.T, testCtx *TestContext) *schedulerframework.Queued
 }
 
 // NextPod returns the next Pod in the scheduler queue, with a 5 seconds timeout.
+// Note that this function leaks goroutines in the case of timeout; even after this function returns after timeout,
```

Comment:
Let's open an issue for this. It should be possible to send a Broadcast to the sync.Cond in the queue.

Reply:
https://github.com/kubernetes/kubernetes/issues/123480

Reply:
Can you revert this so that we don't depend on an owner for this file?

Reply:
/approve

As test owner.